main.go

```diff
@@ -734,10 +741,51 @@ func commentToNotifyUser(args commentArgs) error {
 	if !args.mergeConflicts {
 		commentBody = fmt.Sprintf("@%s I have created a PR for you, ready to merge as soon as tests are passed", args.userName)
 	} else {
-		msg := "@%s I have created a PR for you. Unfortunately there were some merge conflicts " +
-			"which I failed to resolve automatically. Therefore the branch of this PR " +
-			"is ready to be merged locally, and then pushed here :)"
-		commentBody = fmt.Sprintf(msg, args.userName)
+		tmplString := `\
+@{{.UserName}} I have created a PR for you.
+Unfortunately, a merge conflict was detected. This means that the conflict will have to be resolved manually by you, human. Then pushed to the PR-branch, once all conflicts are resolved.
+This can be done by following this recipe:
+
+* Make sure that the 'mender-test-bot', and 'open-source' remotes are present in your repository, if not:
+- git remote add mender-test-bot git@github.com:mender-test-bot/{{.Repo}}-enterprise.git
+- git remote add enterprise     git@github.com:mendersoftware/{{.Repo}}-enterprise.git
```

Comment:
I think it would be safe to assume that origin is the correct repo. I would remove this to simplify instructions and use origin. For advanced users (I do have custom remote names) they will manage to adapt the instructions to their settings.